package.json

```diff
@@ -10,6 +10,7 @@
         "@testing-library/jest-dom": "^4.2.4",
         "@testing-library/react": "^9.3.2",
         "@testing-library/user-event": "^7.1.2",
+        "copy-to-clipboard": "^3.3.1",
```

Comment:
No need to add an extra package just for a minor functionality, I believe the copying functionality can be achieved by using `document.execCommand('copy')`

Reply:
I was not sure about a common place in the project where I could add it so it will be reusable among other components. Where would you suggest?

Reply:
There's a folder called utils, which contains other reusable functions, you can create a function called copyToClipboard and pass the element id as an argument to it. In that function, you can then select the textContent of the element passed and copy it to the clipboard.
Also, do the edge case handling in the function, when the passed id is undefined or the element corresponding to the id doesn't exist.

Reply:
Got it. Will push in some time.

Reply:
Great.

Reply:
@DemonDaddy22 Please check now.

Reply:
There is `navigator.clipboard.writeText` but it does not have support for older browsers.